ICAFIX/functionhighpassandvariancenormalize.m

```diff
@@ -96,21 +102,15 @@
 if dovol > 0
     cts=cts./repmat(Outcts.noise_unst_std,1,ctsT);
 	% Use '_vnts' (volume normalized time series) as the suffix for the volumetric VN'ed TCS
-    if hp>=0
```

Comment:
I believe this change will result in _hp0_hp0, which is not what is wanted.  The distinction here needs to be restored I think.

Reply:
I don't think so. 'hpstring' is defined as an empty string in the hp<0 condition, which eliminates the need to have two separate 'save_avw' commands that only differ in that one included 'hp' as part of the file name (in the hp>=0 condition) and the other one didn't.